man/networkd.conf.xml

```diff
@@ -138,6 +138,13 @@
             </para></listitem>
           </varlistentry>
 
+          <varlistentry>
+            <term><option>custom</option></term>
+            <listitem><para>If <literal>DUIDType=custom</literal>, then the <literal>DUIDRawData=</literal> value will
+            used be as custom identifier. If <literal>DUIDType=custom</literal> is specified then
+            <literal>DUIDRawData=</literal> key is mandatory. Note it applies only on DHCPv6 clients.</para></listitem>
```

Comment:
"then the <literable>..."

also where is this enforced? Shouldn't there be a check if 'custom' is set but DRUIDRawData= is not specified?